src/internal/utils/mdObjectHelper.ts

```diff
@@ -45,3 +47,23 @@ export function isStacked(mdObject: VisualizationObject.IVisualizationObjectCont
 export function hasMeasures(mdObject: VisualizationObject.IVisualizationObjectContent): boolean {
     return mdObject && getMeasuresFromMdObject(mdObject).length > 0;
 }
+
+function isAllMeasuresOnSingleAxis(
+    mdObject: VisualizationObject.IVisualizationObjectContent,
+    secondaryYAxis: IAxisConfig,
+): boolean {
+    const measureCount = getMeasuresFromMdObject(mdObject).length;
+    const numberOfMeasureOnSecondaryAxis = secondaryYAxis ? secondaryYAxis.measures.length : 0;
+    return numberOfMeasureOnSecondaryAxis === 0 || measureCount === numberOfMeasureOnSecondaryAxis;
+}
+
+export function isSimpleStackMeasures(
```

Comment:
it mean that stacked chart without
- dual axes
- group categories

I prefer `areSimpleStackMeasures` as better one.

Reply:
please add some comments (above the function name) about `simple stack measures`

Reply:
I see the purpose of this condition is to ensure `canSortStackTotalValue = true`.
So that it can be used in this `if`
```
if ((viewByAttributeIdentifier && stackByAttributeIdentifier) || canSortStackTotalValue) {...}
```

Furthermore, if we have `simple`, we will also have something like `isComplexStackMeasures` which we should avoid.